pandas/core/frame.py

```diff
@@ -5592,22 +5592,79 @@ def corrwith(self, other, axis=0, drop=False):
 
     def count(self, axis=0, level=None, numeric_only=False):
         """
-        Return Series with number of non-NA/null observations over requested
-        axis. Works with non-floating point data as well (detects NaN and None)
+        Count non-NA cells for each column or row.
+
+        Return Series with number of non-NA observations over requested
+        axis. Works with non-floating point data as well (detects `None`,
+        `NaN` and `NaT`).
 
         Parameters
         ----------
         axis : {0 or 'index', 1 or 'columns'}, default 0
-            0 or 'index' for row-wise, 1 or 'columns' for column-wise
-        level : int or level name, default None
-            If the axis is a MultiIndex (hierarchical), count along a
-            particular level, collapsing into a DataFrame
+            If 0 or 'index' counts are generated for each column.
+            If 1 or 'columns' counts are generated for each **row**.
+        level : int or str, optional
+            If the axis is a `MultiIndex` (hierarchical), count along a
+            particular `level`, collapsing into a `DataFrame`.
+            A `str` specifies the level name.
         numeric_only : boolean, default False
-            Include only float, int, boolean data
+            Include only `float`, `int` or `boolean` data.
 
         Returns
         -------
-        count : Series (or DataFrame if level specified)
+        Series or DataFrame
+            For each column/row the number of non-NA/null entries.
+            If `level` is specified returns a `DataFrame`.
+
+        See Also
+        --------
+        Series.count: number of non-NA elements in a Series
+        DataFrame.shape: number of DataFrame rows and columns (including NA
+            elements)
+        DataFrame.isna: boolean same-sized DataFrame showing places of NA
+            elements
+
+        Examples
+        --------
+        Constructing DataFrame from a dictionary:
+
+        >>> df = pd.DataFrame({"Person":
+        ...                    ["John", "Myla", None, "John", "Myla"],
+        ...                    "Age": [24., np.nan, 21., 33, 26],
```

Comment:
PEP8: indendt one more space. smae with line below.

Reply:
For me flake complains if I change that. on my system flake doesn't check the examples, so I copy it in the code:

```
        df = pd.DataFrame({"Person":
                           ["John", "Myla", None, "John", "Myla"],
                           "Age": [24., np.nan, 21., 33, 26],
                           "Single": [False, True, True, True, False]})
        df
```

If I have it like it like this flake only complains about the pd not being defined.

Reply:
Sorry I misread.